sycl/test/scheduler/MultipleDevices.cpp

```diff
@@ -1,6 +1,6 @@
 // RUN: %clangxx -fsycl %s -o %t.out -lOpenCL
-// RUN: %t.out
-//===- MultipleDevices.cpp - Test checkking multi-device execution --------===//
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
```

Comment:
```c++
try {
    cpu_selector CPUSelector;	    cpu_selector CPUSelector;
    MyQueue1 = queue(CPUSelector);	    MyQueue1 = queue(CPUSelector);
    Result = multidevice_test(MyQueue1, MyQueue2);	    Result = multidevice_test(MyQueue1, MyQueue2);
    MyQueue2 = queue(CPUSelector);	    MyQueue2 = queue(CPUSelector);
    Result |= multidevice_test(MyQueue1, MyQueue2);	    Result |= multidevice_test(MyQueue1, MyQueue2);
  } catch (cl::sycl::invalid_parameter_error &) {	  } catch (cl::sycl::invalid_parameter_error &) {
    std::cout << "Using 2 host devices." << std::endl;	    std::cout << "Using 2 host devices." << std::endl;
  }
}
```
It looks like test is supposed handle the situation when CPU runtime is not available.

Reply:
I think we can replace:
`cpu_selector CPUSelector;
 MyQueue1 = queue(CPUSelector);`

with `TestQueue MyQueue1 {sycl::default_selector()}`,
where TestQueue is defined in helpers.hpp

Reply:
Will give it a try!

Reply:
I think this test was broken some time ago.
Suggest fixing like this:
```
int Result = 1;

try {
  queue MyQueue1(host);
  queue MyQueue2(host);
  Result &= multidevice_test(MyQueue1, MyQueue2);
} catch {
std::cout << "Skipping host and host" << std::endl;
}

try {
  queue MyQueue1 = queue(host);
  queue MyQueue2 = queue(CPUSelector);
  Result &= multidevice_test(MyQueue1, MyQueue2);
} catch {
  std::cout << "Skipping host and CPU " << std::endl;
}

try {
  queue MyQueue1 = queue(CPUSelector);
  queue MyQueue2 = queue(CPUSelector);
  Result &= multidevice_test(MyQueue1, MyQueue2);
} catch {
  std::cout << "skipping cpu and cpu" << std::endl;
}

try {
  queue MyQueue1 = queue(CPUSelector);
  queue MyQueue2 = queue(GPUSelector);
  Result &= multidevice_test(MyQueue1, MyQueue2);
} catch {
  std::cout << "skipping cpu and gpu" << std::endl;
}
...
```
The helper function with takes two device types can be used.

Reply:
Thanks for the great example fix suggestion!